Assets/Scripts/EnemyDamage.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyDamage : MonoBehaviour
+{
+    public PlayerHealth playerHealth;
```

Comment:
Instead of setting this in editor. This could be gotten in the Start mehtod with GameObject.FindGameObjectWithTag and with getcomponent. This way if there are multiple enemies the reference will be automatic and it's not then required to click and drag the playerscript reference to each enemy.